code/modules/mob/living/basic/vermin/cockroach.dm

```diff
@@ -166,3 +166,24 @@
 
 /datum/ai_behavior/basic_melee_attack/sewer
 	action_cooldown = 0.8 SECONDS
+
+/mob/living/basic/cockroach/glockroach/mobroach
+	name = "mobroach"
+	desc = "WE'RE FUCKED, THAT GLOCKROACH HAS A TOMMYGUN!"
+	icon_state = "mobroach"
+	ai_controller = /datum/ai_controller/basic_controller/cockroach/glockroach/mobroach
+/datum/ai_controller/basic_controller/cockroach/glockroach/mobroach
+	planning_subtrees = list(
+    	/datum/ai_planning_subtree/pet_planning,
```

Comment:
These lines have mixed tab/space indentation that's causing failures

Reply:
thank you so much